docs/code-samples/getting-started/daml/Scripts/FixedRateBond.daml

```diff
@@ -0,0 +1,259 @@
+    , tokenFactoryCid
+    , aliceHoldingCid
+    , bobHoldingCid} <- runSettlement
+
+  now <- getTime
+  let pp = [("FactoryProvider", S.singleton public)]
+
+  -- Create and distribute bond
+  -- CREATE_FIXED_RATE_BOND_VARIABLES_BEGIN
+  let
+    issueDate = date 2019 Jan 16
+    firstCouponDate = date 2019 May 15
+    maturityDate = date 2020 May 15
+    notional = 1.0
+    couponRate = 0.011
+    couponPeriod = M
+    couponPeriodMultiplier = 12
+    dayCountConvention = ActActISDA
+    businessDayConvention = Following
+  -- CREATE_FIXED_RATE_BOND_VARIABLES_END
+
+  -- CREATE_FIXED_RATE_BOND_CALENDARS_BEGIN
+  let
+    holidayCalendarIds = ["USD"]
+    cal =
+      HolidayCalendarData with
+        id = "USD"
+        weekend = [Saturday, Sunday]
+        holidays = [date 2019 Dec 19]
+
+  -- A reference data provider publishes the holiday calendar on the ledger
+  calendarCid <- submit bank do
+    createCmd HolidayCalendar with
+      provider = bank
+      calendar = cal
+      observers = M.fromList pp
+  -- CREATE_FIXED_RATE_BOND_CALENDARS_END
+
+  -- CREATE_PERIODIC_SCHEDULE_BEGIN
+  let
+    (y, m, d) = toGregorian firstCouponDate
+    periodicSchedule = PeriodicSchedule with
+      businessDayAdjustment =
+        BusinessDayAdjustment with
+          calendarIds = holidayCalendarIds
+          convention = businessDayConvention
+      effectiveDateBusinessDayAdjustment = None
+      terminationDateBusinessDayAdjustment = None
+      frequency =
+        Periodic Frequency with
+          rollConvention = DOM d
+          period = Period with
+            period = couponPeriod
+            periodMultiplier = couponPeriodMultiplier
+      effectiveDate = issueDate
+      firstRegularPeriodStartDate = Some firstCouponDate
+      lastRegularPeriodEndDate = Some maturityDate
+      stubPeriodType = None
+      terminationDate = maturityDate
+  -- CREATE_PERIODIC_SCHEDULE_END
+
+  -- CREATE_FIXED_RATE_BOND_INSTRUMENT_BEGIN
+  fixedRateBondFactoryCid <- toInterfaceContractId @FixedRate.F <$> submit bank do
+    createCmd FixedRate.Factory with
+      provider = bank
+      observers = M.empty
+
+  -- Define an instrument key for the bond
+  let
+    instrumentId = Id "FixedRateBond"
+    instrumentVersion = "0"
+    bondInstrument = InstrumentKey with
+      issuer = bank
+      depository = bank
+      id = instrumentId
+      version = instrumentVersion
+
+  -- Bank creates the bond instrument
+  fixedRateBondCid <- submit bank do
+    exerciseCmd fixedRateBondFactoryCid FixedRate.Create with
+      fixedRate = FixedRate with
+        instrument = bondInstrument
+        description = "Instrument representing units of a fixed rate bond"
+        couponRate
+        periodicSchedule
+        holidayCalendarIds
+        calendarDataProvider = bank
+        dayCountConvention
+        currency = usdInstrument
+        notional
+        lastEventTimestamp = now
+      observers = M.fromList pp
+  -- CREATE_FIXED_RATE_BOND_INSTRUMENT_END
```

Comment:
Do these new tags replace old ones that should then be removed?

Reply:
Are you referring to the tags in `Daml.Finance.Instrument.Bond.Test.Util`? I was planning on keeping them and using for the new Instruments section (see the doc structure document), which is basically the content of the current instrument tutorials.